packages/harpparts/src/pitch/types/typeguards.test.ts

```diff
@@ -1,7 +1,8 @@
 import { generatePitch } from '../generate-pitch'
 import { PozitionIds } from '../../pozition'
+import { getPitch, getPozition } from '../../access-parts'
 
-import { isPitchId, isNaturalPitch } from './typeguards'
+import { isPitchId, isNaturalPitch, isPitch } from './typeguards'
```

Comment:
Probably should raise a ticket for finishing off this `HarpPart` typing and typeguard process.

Reply:
https://github.com/js-jslog/harpguru/issues/73 raised